DDDEastAnglia/Helpers/Security/IOAuthClientInfo.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.Collections.Generic;
+using DotNetOpenAuth.AspNet;
+using Microsoft.Web.WebPages.OAuth;
+
+namespace DDDEastAnglia.Helpers.Security
+{
+    public interface IOAuthClientInfo
```

Comment:
I agree, but this pull request is already large so I don't think it is the right place to do that.